planet.js

```diff
@@ -753,4 +753,21 @@ planet.connect = async (rn, {online = true} = {}) => {
   return s;
 }; */
 
-// planet.connect('lol');
+const bindConnectButton = () => {
+  const button = document.getElementById('connectButton');
+  if (button) {
+    document.getElementById('connectButton').addEventListener('mouseup', (e) => {
+      e.preventDefault();
+      planet.connect('lol');
+    })
+  } else {
+    setTimeout(() => {
```

Comment:
Why this timeout?

Reply:
waiting for the button to load in the dropdown

Reply:
Why isn't that instant?

Reply:
It looks like the bindings just need to be waiting for window `load` event or something, rather than relying on timeouts.